Source/Data/bin/cook.py

```diff
@@ -135,8 +141,11 @@
             readings = bpmf_phrases[mykey]
         except:
             sys.exit('[ERROR] %s key mismatches.' % mykey)
-        phrases[mykey] = True
-        # print mykey
+        phrases[mykey] = myvalue
+
+    for (mykey, myvalue) in phrases.items():
```

Comment:
nit: `()` is not needed, `for mykey, myvalue in phrases.items():` will work.

Reply:
Done. Thanks!